now.json

```diff
@@ -1,10 +1,68 @@
 {
-    "name": "magento-venia",
-    "alias": ["magento-venia", "veniapwa.com"],
-    "engines": {
-        "node": "10.x.x"
+    "version": 2,
+    "name": "venia",
+    "alias": [
+        "magento-venia-develop"
+    ],
+    "scope": "magento-research",
+    "env": {
+        "UPWARD_JS_BIND_LOCAL": "false",
+        "USE_FASTLY": "1",
+        "MAGENTO_BACKEND_URL": "https://master-7rqtwti-xjmpcdjpe6kzm.us-4.magentosite.cloud/",
+        "NODE_ENV": "production"
```

Comment:
We don't, because that makes Yarn install only production dependencies. We specify `NODE_ENV=production` in the build _script_, but that's after install is complete.